content-review/impl/urkund/pom.xml

```diff
@@ -0,0 +1,202 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<name>Content Review Urkund Implementation</name>
+	<groupId>org.sakaiproject.contentreview.urkund</groupId>
+	<artifactId>content-review-urkund-impl</artifactId>
+	<packaging>jar</packaging>
+
+	<parent>
+		<artifactId>content-review-impl</artifactId>
+		<groupId>org.sakaiproject.contentreview</groupId>
+		<version>12-SNAPSHOT</version>
+	</parent>
+
+	<dependencies>
+		<!-- Content Review dependencies -->
+		<dependency>
+			<groupId>org.sakaiproject.contentreview</groupId>
+			<artifactId>content-review-api</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.sakaiproject.contentreview.dao</groupId>
+			<artifactId>content-review-dao-impl</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.sakaiproject.contentreview.service</groupId>
+			<artifactId>content-review-service-impl</artifactId>
+		</dependency>
+
+		<!-- Assignment dependencies -->
+		<dependency>
+			<groupId>org.sakaiproject.assignment</groupId>
+			<artifactId>sakai-assignment-api</artifactId>
+		</dependency>
+
+		<!-- Sakai Person -->
+		<dependency>
+			<groupId>org.sakaiproject.common</groupId>
+			<artifactId>sakai-common-edu-person-api</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.sakaiproject.common</groupId>
+			<artifactId>sakai-common-manager-api</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.sakaiproject.common</groupId>
+			<artifactId>sakai-common-type-api</artifactId>
+		</dependency>
+
+		<!-- Various Sakai dependencies -->
+		<dependency>
+			<groupId>org.sakaiproject.kernel</groupId>
+			<artifactId>sakai-kernel-api</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.sakaiproject.kernel</groupId>
+			<artifactId>sakai-kernel-util</artifactId>
+		</dependency>
+
+		<dependency>
+			<groupId>org.sakaiproject.kernel</groupId>
+			<artifactId>sakai-component-manager</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.sakaiproject.entitybroker</groupId>
+			<artifactId>entitybroker-api</artifactId>
+		</dependency>
+
+		<!-- generic DAO -->
+		<dependency>
+			<groupId>org.sakaiproject.genericdao</groupId>
```

Comment:
Please don't use genericdao as this lib is deprecated and will be removed in the future.